roles/openshift_provisioners/templates/efs.j2

```diff
@@ -46,6 +46,18 @@ spec:
               value: "{{openshift_provisioners_efs_region}}"
             - name: PROVISIONER_NAME
               value: "{{openshift_provisioners_efs_name}}"
+{% if openshift_provisioners_efs_http_proxy is defined %}
```

Comment:
Why would we want a separate var for EFS? Shouldn't it just be `openshift_http_proxy`?

Reply:
Hi, we have openshift_http_proxy and https_proxy defined but that doesnt work on the EFS provisioners go routine doing a describe efs volume to ensure it exists.

Reply:
Ah, I see. A new var is fine then I guess.

Please make sure its mentioned in `hosts.examples`

Reply:
There is no EFS component mentioned in hosts.examples.